lib/mappers/cdl_json_mapper.py

```diff
@@ -8,18 +8,27 @@ class CDLJSONMapper(MAPV3JSONMapper):
     def __init__(self, provider_data):
         super(CDLJSONMapper, self).__init__(provider_data)
 
+    def map_collection(self):
+        values = iterify(getprop(self.provider_data, "collection_name", True))
+        collections = []
+        for value in values:
+            collection = {"title": value, "@id": "", "id": ""}
+            collections.append(collection)
+
+        self.mapped_data["sourceResource"]["collection"] = collections
```

Comment:
What about 
`if collections: self.mapped_data[...] = collections ` to prevent records from getting an empty array as the collection value.